fossas/spectrometer
# Warn when using older .fossa.yml file

# Overview

This PR, introduces following change:
- Issue Warning when older `.fossa.yml` file (any version below 3) is found for configurations

## Acceptance criteria

- When users are using older (version 1 and version 2) format of `.fossa.yml`, warning is rendered
- Warning includes documentation of `.fossa.yml` specifications 

## Testing plan

- Create `.fossa.yml` with version 2
```
version: 2

serverOP: https://app.fossa.com
apiKey: a1b2c3
```
- Run `fossa analyze -o`, warning should be included, and analysis should still continue

<img width="761" alt="CleanShot 2021-10-05 at 12 15 03@2x" src="https://user-images.githubusercontent.com/86321858/136079709-6dc54123-872c-4391-abd5-c866a2fb8115.png">

## Risks

N/A

## References

Closes - https://github.com/fossas/team-analysis/issues/767

## Checklist
